modules/api/test/LorisApiRecordingsTest.php

```diff
@@ -0,0 +1,1123 @@
+            'string'
+        );
+        $this->assertSame(
+            gettype($recChannelsMetaArray['Electrodes']['0']['Y']),
+            'string'
+        );
+        $this->assertSame(
+            gettype($recChannelsMetaArray['Electrodes']['0']['Z']),
+            'string'
+        );
+        $this->assertSame(
+            gettype($recChannelsMetaArray['Electrodes']['0']['Impedance']),
+            'NULL'
+        );
+        $this->assertSame(
+            gettype($recChannelsMetaArray['Electrodes']['0']['ElectrodeFilePath']),
+            'string'
+        );
+
+        $this->assertArrayHasKey(
+            'Meta',
+            $recChannelsMetaArray
+        );
+        $this->assertArrayHasKey(
+            'CandID',
+            $recChannelsMetaArray['Meta']
+        );
+        $this->assertArrayHasKey(
+            'Visit',
+            $recChannelsMetaArray['Meta']
+        );
+        $this->assertArrayHasKey(
+            'Electrodes',
+            $recChannelsMetaArray
+        );
+        $this->assertArrayHasKey(
+            '0',
+            $recChannelsMetaArray['Electrodes']
+        );
+        $this->assertArrayHasKey(
+            'ElectrodeName',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'ElectrodeType',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'ElectrodeMaterial',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'X',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'Y',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'Z',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'Impedance',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+        $this->assertArrayHasKey(
+            'ElectrodeFilePath',
+            $recChannelsMetaArray['Electrodes']['0']
+        );
+    }
+
+    /**
+     * Tests the HTTP GET request for the
+     * endpoint /candidates/{candid}/{visit}/recordings/{edffile}/events
+     *
+     * @return void
+     */
+    public function testGetCandidatesCandidVisitRecordingsEdfFileEvents(): void
+    {
+        $response = $this->client->request(
+            'GET',
+            "candidates/$this->candidTest/$this->visitTest/recordings/" .
+            "$this->frecordTest/events/meta",
+            [
+                'http_errors' => false,
+                'headers'     => $this->headers
+            ]
+        );
+        if ($response->getStatusCode() === 404) {
+            $this->markTestIncomplete(
+                "Endpoint not found: GET" .
+                "candidates/$this->candidTest/$this->visitTest/recordings/" .
```

Comment:
should fail

Reply:
It exists when the file actually exists. We need to skip to avoid an error, because it is not in the Docker environment. Thus, not skipping causes Travis to fail